pyteal/ast/assert_.py

```diff
@@ -11,23 +12,29 @@
 class Assert(Expr):
     """A control flow expression to verify that a condition is true."""
 
-    def __init__(self, cond: Expr) -> None:
+    def __init__(self, cond: Expr, *condMulti: Expr) -> None:
         """Create an assert statement that raises an error if the condition is false.
 
         Args:
             cond: The condition to check. Must evaluate to a uint64.
         """
         super().__init__()
         require_type(cond, TealType.uint64)
-        self.cond = cond
+        for cond_single in condMulti:
+            require_type(cond_single, TealType.uint64)
+        self.cond = [cond] + list(condMulti)
 
     def __teal__(self, options: "CompileOptions"):
+        if len(self.cond) > 1:
+            asserts = [Assert(cond) for cond in self.cond]
+            return Seq(*asserts).__teal__(options)
+
         if options.version >= Op.assert_.min_version:
             # use assert op if available
-            return TealBlock.FromOp(options, TealOp(self, Op.assert_), self.cond)
+            return TealBlock.FromOp(options, TealOp(self, Op.assert_), self.cond[0])
 
         # if assert op is not available, use branches and err
-        condStart, condEnd = self.cond.__teal__(options)
+        condStart, condEnd = self.cond[0].__teal__(options)
```

Comment:
nit: the version check of `Op.assert_` is not imposed over case `len(self.cond) > 1`.